### DISCUSSION TITLE
Hot reload from terminal

### DISCUSSION THREAD

**Discussion by letto4135** [leoafarias/fvm#804] [Q&A]
Can fvm hot reload from the terminal without having to go to the terminal and hit `r`? Nodemon, Django, and other things are able to auto hot reload from the terminal so not sure why flutter can't, or not sure why I can't figure out how to do it if it can...

**Comment by letto4135** [comment]
Just found this repo
https://github.com/erickzanardo/dashmon

Maybe fvm could integrate it.

**Reply by leoafarias** [reply]
It seems that it does support fvm here https://github.com/erickzanardo/dashmon?tab=readme-ov-file#fvm-support, hot reload should behave just like the normal flutter cli
